shuup/admin/modules/orders/views/edit.py

```diff
@@ -178,8 +178,6 @@ def get_toolbar(self):
     def get_config(self):
         order = self.object
         shop_queryset = Shop.objects.get_for_user(self.request.user).filter(status=ShopStatus.ENABLED)
-        if getattr(self.request.user, "is_superuser", False):
```

Comment:
why removing this? you SHOULD filter shops by current user, otherwise user will see shops that he is not allowed to

Reply:
oh, I see, `get_for_user` does the trick